pytest_html/plugin.py

```diff
@@ -392,7 +393,9 @@ def generate_summary_item(self):
             html.script(raw(main_js)),
             html.p('Report generated on {0} at {1}'.format(
                 generated.strftime('%d-%b-%Y'),
-                generated.strftime('%H:%M:%S'))),
+                generated.strftime('%H:%M:%S')),
+                html.a(' pytest-html', href=__pypi_url__),
```

Comment:
Add ' by ' so it reads 'Report generated on 28-Nov-2016 at 09:17:35 by pytest-html v1.11.1'